integration_tests/e2e/notifications-settings.cy.ts

```diff
@@ -32,7 +32,7 @@ context('A staff member can view their notification settings', () => {
     page.inputEmail().clear()
     page.submit()
 
-    page.errorSummary().contains('Enter your email address')
+    page.errorSummary().contains('Email address cannot be blank')
```

Comment:
following guidance from https://design-system.service.gov.uk/components/radios/#if-it-s-a-conditionally-revealed-question